### ISSUE TITLE
Frequent crashing

### ISSUE THREAD

**Issue by Anti-Widdershins** [Microsoft/vscode#67316]
Issue Type: <b>Bug</b>

Any file or folder open with any number of windows.
App will crash frequently when window is brought to front, or not touched at all for a few minutes.
Close to unusable as the last open files are not remembered but a previous set is remembered for some reason.

VS Code version: Code 1.30.2 (61122f88f0bf01e2ac16bdb9e1bc4571755f5bd8, 2019-01-07T22:54:13.295Z)
OS version: Windows_NT x64 10.0.17763

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Xeon(R) CPU E5-2687W v2 @ 3.40GHz (32 x 3392)|
|GPU Status|2d_canvas: unavailable_software<br>checker_imaging: disabled_off<br>flash_3d: unavailable_software<br>flash_stage3d: unavailable_software<br>flash_stage3d_baseline: unavailable_software<br>gpu_compositing: unavailable_software<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: unavailable_software<br>video_decode: unavailable_software<br>video_encode: unavailable_software<br>webgl: enabled_readback<br>webgl2: unavailable_off|
|Memory (System)|63.93GB (50.70GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (7)</summary>

Extension|Author (truncated)|Version
---|---|---
code-runner|for|0.9.6
MagicPython|mag|1.1.0
python|ms-|2018.12.1
cpptools|ms-|0.21.0
csharp|ms-|1.17.1
PowerShell|ms-|1.11.0
vscode-icons|rob|8.0.0

</details>

**Comment by frank-dspeed** [comment]
@Anti-Widdershins it could be a electron bug can you verify that via using the insider build if that don't crashes its the elctron bug that is knowed already:

https://code.visualstudio.com/insiders/

**Comment by Anti-Widdershins** [comment]
Thanks for the tip, but the insiders version is also crashing in the same way, albeit, apparently less often.

**Comment by frank-dspeed** [comment]
Ok then @Tyriar needs to jump in i only saw the issue and was thinking that this is the current elctron bug but it looks like its not the bug that i meand

**Comment by Tyriar** [comment]
Could you see if it reproduces if you close all windows and launch with `code --disable-extensions --verbose`? If it happens paste in the output of the command here.

**Comment by Anti-Widdershins** [comment]
I invoked code with the command line options that you suggested. I had lots of other (non-code) windows open at the time. No crashing has occurred so far. I'll keep experimenting with it.
Thanks for your help.

**Comment by frank-dspeed** [comment]
@Anti-Widdershins so its a extension issue

**Comment by Anti-Widdershins** [comment]
It looks like my installation of VSCode (basic) was corrupted somehow. I gradually deleted all extensions but the C/C++ extension without having any effect on the crashing. At that point the extensions window hung up trying to connect to the web. I could not change it so I uninstalled VSCode. Interestingly, since then the VSCode-insiders version stopped crashing - some cross effect?
Trying to download the VSCode basic installer was fraught with difficulty because it was always corrupted (with antivirus turned off) regardless of which binary installation I tried to fetch. On the 12th attempt I got a clean download.
I've installed it and added two extensions and it doesn't crash.

Thanks very much for your assistance.
